.github/workflows/trigger-builds.yaml

```diff
@@ -0,0 +1,52 @@
+name: Trigger workflow for release
+
+on:
+  workflow_dispatch:
+    inputs:
+      cache-manager-ref:
+        type: string
+        default: main
+      cache-manager-repository:
+        type: string
+        default: gingersnap-project/cache-manager
+      db-syncer-ref:
+        type: string
+        default: main
+      db-syncer-repository:
+        type: string
+        default: gingersnap-project/db-syncer
+      operator-ref:
+        type: string
+        default: main
+      operator-repository:
+        type: string
+        default: gingersnap-project/operator
+      tag:
+         type: string
+         default: latest
+
+jobs:
+
+  cache-manager-images:
+    uses: gingersnap-project/cache-manager/.github/workflows/create-tag.yml@main
+    with:
+      ref: ${{ inputs.cache-manager-ref }}
+      repository: ${{ inputs.cache-manager-repository }}
+      tag: ${{ inputs.tag }}
+
+  db-syncer-image:
+    uses: gingersnap-project/db-syncer/.github/workflows/create-tag.yml@main
```

Comment:
Needs updating to `publish-release.yaml`.

Reply:
@${{ inputs.cache-manager-ref }} for the @ version this is not possible it seems . It giving error "version cannot have whitespace"